packages/fx-core/src/common/samples-config-v3.json

```
@@ -185,6 +185,16 @@
             "time": "5min to run",
             "configuration": "Ready for debug",
             "suggested": false
+        },
+        {
+          "id": "hello-world-teams-tab-and-outlook-add-in",
+          "title": "Hello World Teams tab and Outlook add-in",
```

Better to follow the title-style capitalization rule:
```suggestion
          "title": "Hello World Teams Tab and Outlook Add-in",
```

---

Changed "tab" to "Tab". Outlook add-in with lower-case "a" is the term to use in all docs.